forum/Sources/Tips.php

```diff
@@ -0,0 +1,81 @@
+<?php 
+
+if (!defined('SMF'))
+	die('Hacking attempt...');
+
+function Tips()
+{
+	global $context, $smcFunc, $user_info, $txt;
+
+	$context['page_title'] = $txt['tip_list_title'];
+	
+	loadPosts();
+
+	loadTemplate('Tips');
+}
+
+function loadPosts()
+{
+	global $memberContext, $context, $smcFunc, $user_info;
+	
+	$tippedPosts = array();
+	
+	// Find all tips in descending order
+	// TO-DO: custom-limit: pagination support, etc.
+	// Get most recent tipped posts
+	$query = 'SELECT id_message_tip, id_message, id_member, coins FROM {db_prefix}message_tips';

```

Comment:
Is `{db_prefix}` within queries a feature of SMF?

Reply:
Yup